app/konnect/gateway-manager/plugins/add-custom-plugin.md

```diff
@@ -17,7 +17,8 @@ See [Editing or deleting a custom plugin's schema](/konnect/gateway-manager/plug
 for more information.
 
 {:.note}
-> **Note**: Custom plugins are not supported in Dedicated Cloud Gateways.
+> **Note**: For adding custom plugins to a Dedicated Cloud Gateway, see 
+[Custom plugins in Dedicated Cloud Gateways](/konnect/gateway-manager/dedicated-cloud-gateways/custom-plugins/).
```

Comment:
Nice to have: for Dev site, I think the DCG custom plugin section could be an include that we'd use here instead of the note. But I think the way you have it now it okay for this release.